### PR TITLE
Rewards v2 Testnet: "Rewards" Section

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

File: docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md [hunk 1]
```diff
@@ -9,27 +9,43 @@ title: Rewards Claiming Overview
...
 
-Reward Earners (Stakers and Operators) can set a claimer address that can claim rewards for the tokens they've earned. An Earner is its own claimer by default and only the claimer address can claim rewards. If a new claimer is set, the new address can claim all of the previously unclaimed rewards. The earner can always configure their designated claimer address.
+## Rewards v2 (currently in Tesnet)
+
+Rewards v2 addresses key challenges identified in the EigenLayer Rewards MVP (v1), particularly the need for operator-directed rewards via external logic and flexibility for operators in setting different fee rates to better cover operating costs or attract more stake when running different AVS types. Rewards v2 adds the following use cases:
+- Operator-directed rewards from AVSs, allowing AVSs to segment and reward specific operators. This gives AVSs the flexibility to set custom logic for rewards to individual operators, based on work completed or anything else they may design or desire (e.g., more equal distribution of operator support for decentralization or security reasons).
```
Comment (wesfloyd) [inline docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md:20]: todo, add explanation for Variable Operator Fees  and Operator Splits for Programmatic Incentives

File: docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md [hunk 2]
```diff
@@ -7,29 +7,44 @@ title: Rewards Claiming Overview
 
 ## Overview
 
-The EigenLayer Rewards protocol enables AVSs to make rewards to stakers and operators. Operators earn rewards by opting in to AVSs that make `RewardsSubmissions` to the `RewardsCoordinator`, a core protocol contract. Within a single `RewardsSubmission`, an AVS can specify a time range for which the reward will be distributed, a list of weights for each `Strategy` for the reward, and an ERC20 token to make rewards in.  
+The EigenLayer Rewards protocol enables AVSs to make rewards to stakers and operators. Operators earn rewards by opting in to AVSs that make `RewardsSubmissions` to the `RewardsCoordinator`, a core protocol contract. Within a single `RewardsSubmission`,  an AVS can specify a time range for which the reward will be distributed, a list of weights for each `Strategy` for the reward, and an ERC20 token to make rewards in.
 
-Operators will earn a flat 10% commission on rewards. The rest of the reward is passed on to the operator's delegated stakers. Rewards are proportional to:
+By default, Operators will earn a flat 10% split on rewards. The rest of the reward is claimable bythe operator's delegated Stakers. Rewards are proportional to:
```
Comment (non-fungible-nelson) [inline docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md:12]: by the ---- need a space

File: docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md [hunk 3]
```diff
@@ -7,29 +7,44 @@ title: Rewards Claiming Overview
...
+Not to be confused with the designated claimer address specified above, the recipient address is the address that will receive the ERC20 token rewards. The designated claimer (or the Earner themselves) has the ability to call `RewardsCoordinator.processClaim` for the Earner while also specifying a recipient address to receive all the rewards.
 
+## Reward Calculations
+
+Rewards are calculated via an offchain process. A Merkle root is posted which represents the cumulative rewards across all earners weekly on Mainnet and daily on Testnet. There is an additional 2 hour delay on testnet and 1 week delay on mainnet after posting in order for the root to be claimable against with a valid Merkle proof. The deterministic calculation of the distribution of rewards is specified in our [technical docs](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/docs/core/RewardsCoordinator.md).
```
Comment (non-fungible-nelson) [inline docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md:45]: should we link to sidecar here?

Comment (wesfloyd) [reply docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md:45]: Not yet. The docs are in the works.

File: docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md (hunk 2 above)
Comment (wesfloyd) [reply docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md:12]: Can you clarify what you mean?

Comment (non-fungible-nelson) [reply docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md:12]: there is a typo
between the words by and the we need a space on this line

Comment (wesfloyd) [reply docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md]: good call, resolved now

File: docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md (hunk 3 above)
Comment (wesfloyd) [reply docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md:45]: Note sent to you @non-fungible-nelson  via Slack on this issue

**Pull request merged**